packages/infra/lib/api-stack.ts

```diff
@@ -732,11 +744,22 @@ export class APIStack extends Stack {
       alarmSnsAction: alarmAction,
     });
 
-    // Grant lambda access to the api server
-    server.service.connections.allowFrom(fitbitLambda, Port.allTcp());
+    // grant fitbitSubscriberVerificationLambda read access to the verification code
+    const fitbitVerificationSecret: Secrets = {};
+    buildSecrets(fitbitVerificationSecret, this, {
+      fitbitSubscriberVerificationCode:
+        config.providerSecretNames.FITBIT_SUBSCRIBER_VERIFICATION_CODE,
+    });
+    for (const secret of Object.values(fitbitVerificationSecret)) {
+      secret.grantRead(fitbitSubscriberVerificationLambda);
+    }
+
+    // Grant lambdas access to the api server
+    server.service.connections.allowFrom(fitbitAuthLambda, Port.allTcp());
+    server.service.connections.allowFrom(fitbitSubscriberVerificationLambda, Port.allTcp());
```

Comment:
I think we don't need those, we opened access to any IP within the VPC to the server ([code](https://vscode.dev/github/metriport/metriport/blob/704-rename-migrations/packages/infra/lib/api-service.ts#L177), [ticket to fix](https://github.com/metriport/metriport-internal/issues/489)).

In theory, this would be good b/c then we don't need to fix it later, but b/c its already open for all IPs, we don't know if this is really working or not - so IMO might as well remove the code.

There are other places that we didn't specify access b/c of that.